reviewpad.yml

```diff
@@ -100,3 +100,10 @@ workflows:
   #     - rule: check-for-status-block
   #   then:
   #     - '$addLabel("🚧 status: blocked")'
+  
+  - name: mark-PR-blocked-if-not-status-ready-for-dev
+    always-run: true
+    if: 
+      - '$isElementOf("🏁 status: ready for dev", $labels()) == false'
```

Comment:
I think this will check the PR labels rather than the issue labels https://docs.reviewpad.com/guides/built-ins/#labels

Reply:
Yep, this check pull request labels and not issue labels
There is no option to check the linked issue label at opening the pull request in reviewpad
Reviewpad only allows checking whether opened pull request is linked to any issue

To check issue labels I can look for solutions using GitHub API and GitHub Action